platform/platform_unix_impl.cpp

```diff
@@ -155,6 +155,16 @@ bool Platform::IsFileExistsByFullPath(string const & filePath)
 //static
 void Platform::DisableBackupForFile(string const & filePath) {}
 
+// static
```

Comment:
The order should match the order in .hpp.

Reply:
Fixed.